python/lsst/meas/extensions/gaap/__init__.py

```diff
@@ -20,5 +20,4 @@
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 
-from .version import *
 from .gaap import *
```

Comment:
When you use the sqrbot to create a new python-only package it uses this template: 
https://github.com/lsst/templates/tree/master/project_templates/stack_package/example_pythononly. 
Make sure that you can clone a fresh copy and build it. 
```
$ git clone git@github.com:lsst-dm/meas_extensions_gaap
$ cd  meas_extensions_gaap
$ git checkout tickets/DM-24737
$ scons 
``` 
because I couldn't:
```
(lsst-scipipe) [yusra@lsst-devl01 meas_extensions_gaap]$ scons -j 8
scons: Reading SConscript files ...
EUPS integration: enabled
Setting up environment to build package 'meas_extensions_gaap'.
Failed to load primary package configuration for meas_extensions_gaap.
```

Reply:
I had deleted the .cfg file in ups/ directory, but I had not set `noCfgFile` to True in SConstruct. I confirmed that I was able to clone a fresh copy and build successfully.